aclopez/TemplateStringJs/readme.md

```diff
@@ -0,0 +1,214 @@
+
+![Header](images/header.jpg)
+
+# Templates String in Javascript
+
+"Primero resuelve el problema. Entonces, escribe el código" John Johnson
```

Comment:
Use "> quote"

Reply:
I prefer without > because in Ghost the first paragrahp have a big letter than the others paragraphs and I prefer this style.